blog/_posts/2016-07-01-operation-code-provides-career-opportunities-for-military-veterans.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title:  Operation Code Provides Career Opportunities for Military Veterans
+date:   2016-07-01
+tags:   News
+author: Susy Raybon
+layout: post
+---
+Veterans have been suffering from high unemployment rates for years, but the strategies designed to employee these veterans haven’t changed. The [most current numbers](http://www.usnews.com/news/articles/2014/11/10/private-sector-tackles-veteran-unemployment)** **were released to media sources in late 2014. Force reduction has hit all branches of the military. The [Army alone has plans to reduce its numbers by 200,000 Soldiers,](http://www.armytimes.com/story/military/careers/army/2014/12/26/2015-drawdown-year-ahead/20860491/) which, no doubt, will lead to increased unemployment for veterans.
```

Comment:
Something weird's going on with the formatting here. I think we can do without the bold text for this part.